### ISSUE TITLE
prediction.py _get_current_state_dict 重复字段与向后兼容散落

### ISSUE THREAD

**Issue by wuxixixi** [wuxixixi/ProjectInsight#156]
## 问题

`backend/routers/prediction.py` 的 `_get_current_state_dict` 包含重复的向后兼容字段映射，且此模式在多个文件中重复出现。

## 具体位置

- `prediction.py:18-28` — 重复字段映射

## 重复代码

```python
def _get_current_state_dict() -> dict:
    return {
        "negative_belief_rate": engine.current_state.rumor_spread_rate,  # 旧名
        "rumor_spread_rate": engine.current_state.rumor_spread_rate,     # 新名
        "correct_belief_rate": engine.current_state.truth_acceptance_rate, # 旧名
        "truth_acceptance_rate": engine.current_state.truth_acceptance_rate, # 新名
        ...
    }
```

## 影响分析

- 旧字段名和新字段名同时存在，增加维护负担
- 向后兼容逻辑散落在 helpers.py、simulation.py、prediction.py 等多处
- 轨迹接口 `get_prediction_trajectory` 还手动 `pop("silence_rate")`，与其他接口不一致

## 建议修复

1. 在 `models/schemas.py` 或 `helpers.py` 中统一字段映射表
2. 定义 `normalize_state_dict(raw_state)` 函数处理所有兼容性映射
3. 所有路由统一调用此函数
4. 设定旧字段名废弃时间表，逐步移除

## 优先级

低 — 代码一致性与可维护性
